ReactWindows/ReactNative.Shared/ReactRootView.cs

```diff
@@ -47,6 +49,17 @@ internal TouchHandler TouchHandler
         }
 
         /// <summary>
+        /// Get the initialProps
```

Comment:
internal is one of those edge cases that can either be documented or not.  I'd say go ahead and document it, but try to come up with a better comment than `The initialProps.`  Maybe something like "The initial props that are supplied in the `AppRegistry.registerCompoent` callback.

---
In reply to: [97259811](https://github.com/ReactWindows/react-native-windows/pull/961#discussion_r97259811) [](ancestors = 97259811)